plugins/modules/lambda_event.py

```diff
@@ -144,10 +144,109 @@
 
 RETURN = r"""
 ---
-lambda_stream_events:
-    description: list of dictionaries returned by the API describing stream event mappings
+events:
+    description: Dictionary returned by the API describing stream event mappings
     returned: success
-    type: list
+    type: dict
+    contains:
+      batch_size:
+        description: The maximum number of records in each batch that Lambda pulls.
+        type: int
+        returned: always
+      bisect_batch_on_function_error:
+        description: If the function returns an error, split the batch in two and retry.
+        type: bool
+        returned: for Kinesis and DynamoDB Streams only
+        sample: false
+      destination_config:
+        description: Configuration object that specifies the destination of an event after Lambda processes it.
+        type: dict
+        returned: For Kinesis, DynamoDB Streams, Amazon MSK, and self-managed Kafka only
+        contains:
+          on_failure:
+            description: The destination configuration for failed invocations.
+            type: dict
+            contains:
+              destination:
+                description: Destination resource ARN.
+                type: str
+          on_success:
+            description: The destination configuration for failed invocations.
+            type: dict
+            contains:
+              destination:
+                description: Destination resource ARN.
+                type: str
+      event_source_arn:
+        description: The Amazon Resource Name (ARN) of the event source.
+        type: str
+        returned: always
+        sample: "arn:aws:dynamodb:us-west-2:xxxxxx:table/yyyy/stream/2024-06-06T07:03:58.956"
```

Comment:
I'm not sure if we have a practice around example ARNs, but I'd suggest using `123456789012` for the account number.

Reply:
I don't think I documented it properly, but I went through and cleaned up a lot of things like that, `123456789012` is the example account I was using.  (Cleaning up the examples helped keep InfoSec off my back with false positives of "leaked" credentials/resource IDs).  https://github.com/ansible-collections/amazon.aws/pull/1070/files.

I should probably get around to fixing the lack of documentation...